src/support.cc

```diff
@@ -1723,6 +1897,8 @@ bool certifier::utilities::ECC_to_key(const EC_KEY* ecc_key, key_message* k) {
     } else if (modulus_size == 32) {
       k->set_key_type("ecc-256-private");
     } else {
+      printf("%s() error, line: %d, EC_KEY_get0_private_key failed\n",
```

Comment:
? Is the term `EC_KEY_get0_private_key` the right one in the message? 

Should we instead be reporting an invalid modulus size, and report the `modulus_size` value? (Like it's been done elsewhere?)